tests/test_network.py

```diff
@@ -0,0 +1,44 @@
+from autowebcompat import network
+from keras import backend as K
+import math
+import numpy as np
+
+arr1 = np.array([[1, 1]], dtype=np.float32)
+arr2 = np.array([[0, 1]], dtype=np.float32)
+
+arr3 = np.array([[0, 0]], dtype=np.float32)
+arr4 = np.array([[3, 2]], dtype=np.float32)
+
+
+def test_eucledian_distance():
+    dist = network.euclidean_distance([arr1, arr2])
+    evaluate = K.eval(dist)
+    dist2 = network.euclidean_distance([arr3, arr4])
+    evaluate2 = K.eval(dist2)
+    assert (evaluate == 1)
+    assert (math.floor(evaluate2 * 100) == 360)
+
+
+def test_eucl_distance_output_shape():
+    vect = [arr1.shape, arr2.shape]
+    shape = network.eucl_dist_output_shape(vect)
+    assert (shape == (1, 1))
+
+
+def test_contrastive_loss():
```

Comment:
This seems overly complex. As a unit test, this should just test the `contrastive_loss` function.